### ISSUE TITLE
W3C Validation and accessibility checks failed

### ISSUE THREAD

**Issue by d1nuc0m** [qub1750ul/hugo-simpleIntro#5]
[W3C Validation Tool](https://validator.w3.org) will report various errors on a website built with this theme. You can try using _https://gangjun06.github.io/SimpleIntroDemo/_ as a example.

Also assorted accessibility checks are failed as well, such as WCAG check through [MAUVE++](https://mauve.isti.cnr.it/index.jsp) (by CNR)

_Issue part of a series originating from a chat with @zekromaster_
